packages/flame/lib/src/effects/color_effect.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:ui';
+
+import 'package:flutter/animation.dart';
+
+import '../components/mixins/has_paint.dart';
+import 'effects.dart';
+
+class ColorEffect extends ComponentEffect<HasPaint> {
+  final Color color;
+  final double duration;
+  final String? paintId;
+
+  late Paint _original;
+
+  late ColorTween _tween;
+
+  ColorEffect({
+    required this.color,
+    required this.duration,
```

Comment:
Should we have `speed` here too, like the other effects? Maybe it doesn't make sense to express it as `speed`.

Reply:
Yeah, I thought on using speed here, but I can't visualize how to use it on these kind o effects, I think speed is just the result of the difference of color by the duration.

But if you have a good idea on how to use speed here, let me know and I can change

Reply:
Can we resolve this @spydon ?

Reply:
For sure!